gravitee-apim-console-webui/src/management/api/failover-v4/api-failover-v4.component.html

```diff
@@ -0,0 +1,141 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form *ngIf="failoverForm" [formGroup]="failoverForm" autocomplete="off" gioFormFocusInvalid>
+  <mat-card>
+    <mat-card-content>
+      <gio-form-slide-toggle class="failover-card__enable-toggle">
+        <gio-form-label>Enable Failover</gio-form-label>
+        Automatically redirects request to the next endpoint if the response is too slow.
+        <mat-slide-toggle
+          gioFormSlideToggle
+          formControlName="enabled"
+          aria-label="Failover enable toggle"
+          name="enableFailover"
+        ></mat-slide-toggle>
+      </gio-form-slide-toggle>
+
+      <gio-banner-info class="failover-card__info">
+        Failover mechanism operates using a circuit breaker system. When a certain threshold of slow calls or connection failures (ssl
+        error, unknown host, ...) is reached, the circuit breaker reaches "open" state, stopping all requests to the backend. During this
+        time, the API will answer with a 502 - Bad Gateway status.
+        <div class="failover-card__info__button">
+          <button type="button" mat-stroked-button (click)="onMoreInfoClick($event)">
+            <mat-icon svgIcon="gio:eye-empty"></mat-icon>
+            More information
+          </button>
+        </div>
+      </gio-banner-info>
+
+      <mat-divider></mat-divider>
+
+      <div class="failover-card__control" [class.disabled]="!failoverForm.controls.enabled.value">
```

Comment:
Shouldn't we simply disabled the field programmatically ? I'm suprised to see that you need to apply a custom style in this form 🤔